prboom2/src/dsda/global.c

```diff
@@ -609,31 +610,73 @@ static void dsda_InitHexen(void) {
   }
 }
 
+static dboolean dsda_WadMatches(const char* value, const char* base)
+{
+  const char* start;
+  const char* dot;
+  const char* fs;
+  const char* bs;
+  size_t len;
+  size_t baselen;
+
+  baselen = strlen(base);
+
+  // Find final path component
```

Comment:
Isn't this mostly a dup of https://github.com/kraflab/dsda-doom/blob/908f0135b0eb9614abe7f9b71ae04650a3032235/prboom2/src/d_main.c#L1251-L1263 ?

Reply:
Yeah, except for the extension removal (which catches `-iwad heretic`, etc.).  I'm not familiar with the code base so I didn't know that existed.

Reply:
If you want the function moved to a non-static, where should it go?